Help/FAQ.rst

```diff
@@ -8,23 +8,15 @@ What does coala do (for me)?
 ----------------------------
 
 coala is like a spell and grammar checker for source code. Imagine using
-LibreOffice Writer to spellcheck something in English. Why should you have to
-write LibreOfficES to check something written in Spanish?
-That is what coala  tries to fix. Why write the whole thing again if you
-actually just want another analyzing routine? Thats what bears are for. You
-want to use clang or pylint on your project? We got you covered. One command
+LibreOffice Writer to spellcheck something in English but to spellcheck
+something in Spanish con't be done in same. That is what coala tries to fix.
+Just use bears for another analyzing routine, without having to write it
+again. You want to use clang or pylint on your project, just one command
 and one configuration to lint all languages in your project.
 
-You have an awesome idea for a new kind of code analysis but don't want to
-write a CLI? Write just the parameters your custom analysis needs and the
-analysis part, we'll take care of the rest. Everything is handled by coala.
-
-All in all coala does two things:
-
-- Make it easy to use existing static code analyzers by unifying and
-  simplifying the configs
-- Make it easy to write new routines by offering the interface part
-  and everything but the actual analyzer routine.
+If you have an awesome idea for a new kind of code analysis, just write the
+parameters your custom analysis needs and the analysis part, coala will take
```

Comment:
IMO, this can be shortened to `just write the parameters your custom analysis needs and coala will take care of the analysis part.`